core/src/main/resources/hudson/model/Messages_zh_CN.properties

```diff
@@ -35,10 +95,149 @@ BallColor.Pending=\u7B49\u5F85
+BuildAuthorizationToken.InvalidTokenProvided=\u63d0\u4f9b\u7684token\u975e\u6cd5\u3002
+
+#清除构建队列。
+CLI.clear-queue.shortDescription=\u6e05\u9664\u6784\u5efa\u961f\u5217\u3002
+#继续使用节点执行构建任务，替代早期的命令"offline-node"。
+CLI.online-node.shortDescription=\u7ee7\u7eed\u4f7f\u7528\u8282\u70b9\u6267\u884c\u6784\u5efa\u4efb\u52a1\uff0c\u66ff\u4ee3\u65e9\u671f\u7684\u547d\u4ee4"offline-node"\u3002
+
+#代理
+Computer.Caption=\u4ee3\u7406 {0}
+#代理 "{0}" 不存在。可能是"{1}"?
+Computer.NoSuchSlaveExists=\u4ee3\u7406 "{0}" \u4e0d\u5b58\u5728\u3002\u53ef\u80fd\u662f "{1}"?
+#代理 "{0}" 不存在。
+Computer.NoSuchSlaveExistsWithoutAdvice=\u4ee3\u7406 "{0}" \u4e0d\u5b58\u5728\u3002
+#代理
+Computer.Permissions.Title=\u4ee3\u7406
+#该权限允许用户读取代理配置。
+Computer.ExtendedReadPermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u8bfb\u53d6\u4ee3\u7406\u914d\u7f6e\u3002
+#该权限允许用户配置代理。
+Computer.ConfigurePermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u914d\u7f6e\u4ee3\u7406\u3002
+#该权限允许用户删除已存在的代理。
+Computer.DeletePermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u5220\u9664\u5df2\u5b58\u5728\u7684\u4ee3\u7406\u3002
+#该权限允许用户创建代理。
+Computer.CreatePermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u521b\u5efa\u4ee3\u7406\u3002
+#该权限允许用户连接代理或者让代理上线。
+Computer.ConnectPermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u8fde\u63a5\u4ee3\u7406\u6216\u8005\u8ba9\u4ee3\u7406\u4e0a\u7ebf\u3002
+#该权限允许用户断开或者临时下线代理。
+Computer.DisconnectPermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u65ad\u5f00\u6216\u8005\u4e34\u65f6\u4e0b\u7ebf\u4ee3\u7406\u3002
+#该权限允许用户在代理上允许任务。
+Computer.BuildPermission.Description=\u8be5\u6743\u9650\u5141\u8bb8\u7528\u6237\u5728\u4ee3\u7406\u4e0a\u5141\u8bb8\u4efb\u52a1\u3002
+#代理节点离线或者不是一个远程通道（例如：master节点）。
+Computer.BadChannel=\u4ee3\u7406\u8282\u70b9\u79bb\u7ebf\u6216\u8005\u4e0d\u662f\u4e00\u4e2a\u8fdc\u7a0b\u901a\u9053\uff08\u4f8b\u5982\uff1amaster\u8282\u70b9\uff09\u3002
+
+#没有这样的代理
+ComputerSet.NoSuchSlave=\u6ca1\u6709\u8fd9\u6837\u7684\u4ee3\u7406: {0}
+#代理\u2018{0}\u2019已经存在
+ComputerSet.SlaveAlreadyExists=\u4ee3\u7406 \u2018{0}\u2019 \u5df2\u7ecf\u5b58\u5728
+#指定要拷贝的代理
+ComputerSet.SpecifySlaveToCopy=\u6307\u5b9a\u8981\u62f7\u8d1d\u7684\u4ee3\u7406
+#节点列表
+ComputerSet.DisplayName=\u8282\u70b9\u5217\u8868
+
+#(从 <a href="{1}">{0}</a>)
+Descriptor.From=(\u4ece <a href="{1}">{0}</a>)
+
+#不可用
+Executor.NotAvailable=\u4e0d\u53ef\u7528
+
+FreeStyleProject.DisplayName=\u6784\u5EFA\u4E00\u4E2A\u81EA\u7531\u98CE\u683C\u7684\u8F6F\u4EF6\u9879\u76EE
+FreeStyleProject.Description=\u8FD9\u662FJenkins\u7684\u4E3B\u8981\u529F\u80FD.Jenkins\u5C06\u4F1A\u7ED3\u5408\u4EFB\u4F55SCM\u548C\u4EFB\u4F55\u6784\u5EFA\u7CFB\u7EDF\u6765\u6784\u5EFA\u4F60\u7684\u9879\u76EE, \u751A\u81F3\u53EF\u4EE5\u6784\u5EFA\u8F6F\u4EF6\u4EE5\u5916\u7684\u7CFB\u7EDF.
+
+HealthReport.EmptyString=
+
+Hudson.BadPortNumber=\u7aef\u53e3\u53f7\u9519\u8bef {0}
+Hudson.Computer.Caption=Master
+Hudson.Computer.DisplayName=master
```

Comment:
Not required (as well as some entries around)

Reply:
It's the same situation with above.